src/main/java/gtPlusPlus/xmod/gregtech/api/interfaces/internal/IGregtech_RecipeAdder.java

```diff
@@ -336,8 +374,10 @@ boolean addColdTrapRecipe(int aCircuit, ItemStack aInput, FluidStack aFluidInput
     boolean addReactorProcessingUnitRecipe(ItemStack aInput1, ItemStack aInput2, FluidStack aFluidInput,
             ItemStack[] aOutputs, int[] aChances, FluidStack aFluidOutput, int aTime, int aEU);
 
+    @Deprecated
     boolean addFluidHeaterRecipe(ItemStack aCircuit, FluidStack aInput, FluidStack aOutput, int aDuration, int aEUt);
 
+    @Deprecated
     boolean addVacuumFreezerRecipe(ItemStack aInput, ItemStack aOutput, int aDuration, int aEU);
 
     boolean addMolecularTransformerRecipe(ItemStack aInput, ItemStack aOutput, int aDuration, int aEU);
```

Comment:
you missed one

Reply:
molecular transformer multi is a gtpp recipe map. I dont think those have RA2. correct me if I am wrong